src/ServiceControl.Audit.Persistence.RavenDB/MemoryInformationRetriever.cs

```diff
@@ -0,0 +1,32 @@
+namespace ServiceControl.Audit.Persistence.RavenDB;
+
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+class MemoryInformationRetriever(string serverUrl)
+{
+    readonly HttpClient client = new() { BaseAddress = new Uri(serverUrl) };
+
+    record ResponseDto
+    {
+        public MemoryInformation MemoryInformation { get; set; }
+    }
+
+    record MemoryInformation
+    {
+        public bool IsHighDirty { get; set; }
+        public string DirtyMemory { get; set; }
+    }
+
+    public async Task<(bool IsHighDirty, int DirtyMemory)> GetMemoryInformation(CancellationToken cancellationToken = default)
```

Comment:
I would include the size in the name:
```suggestion
    public async Task<(bool IsHighDirty, int DirtyMemoryKB)> GetMemoryInformation(CancellationToken cancellationToken = default)
```